app/src/main/java/br/org/cesar/discordtime/stickysessions/domain/model/NoteFilter.java

```diff
@@ -0,0 +1,12 @@
+package br.org.cesar.discordtime.stickysessions.domain.model;
+
+public class NoteFilter {
+
+    public String mUser;
```

Comment:
I'd rather not use the "m" prefix on public properties like that since you force everyone using `NoteFilter` to keep referring to them as "mUser" and "mIdSession" instead of just "user" or "sessionId".